caption_core.py

```
@@ -0,0 +1,280 @@
+        current_index_str = f"[{i+1}/{len(image_paths)}]"
+
+        if output_path.is_file() and overwrite_checker and not overwrite_checker(output_path):
+            core_log_gui(_get_string_internal("TaggerCore", "Tag_Skipped_Existing_File_Short", current_index_str=current_index_str, output_path_name=output_path.name), "orange")
+            continue
+
+        core_log_gui(_get_string_internal("TaggerCore", "Processing_Image", current_index_str=current_index_str, relative_path=str(relative_path)), "black")
+
+        try:
+            with open(image_path, "rb") as f:
+                image = Image.open(f).convert("RGB")
+        except Exception as e:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)